frontend/src/data/tasks.json

```diff
@@ -1287,7 +1269,7 @@
       {
         "answerKey": "outside-canada",
         "descriptionEn": "Spending time outside Canada may change the way you collect your Old Age Security and Canada Pension Plan (CPP) payments.\n\nYou can continue to collect your government pensions abroad, but the Government of Canada may withhold taxes that could be up to 25%.\n\nIf you are living outside Canada for more than 6 months and collecting Guaranteed Income Supplement (GIS), you will stop collecting your benefit until you return.\n\nYou cannot collect the Allowance for the Survivor benefit if you are outside of Canada for more than 6 months.\n\nIf you plan to leave Canada for more than 6 months, you must [contact Service Canada](https://www.canada.ca/en/contact/contact-1-800-o-canada.html) to avoid an overpayment.",
-        "descriptionFr": "Le fait de passer du temps à l'étranger peut avoir des conséquences sur les prestations de la Sécurité de la vieillesse et du Régime de pensions du Canada (RPC) que vous recevez.\n\nVous pouvez continuer à recevoir vos pensions de l'État à l'étranger, mais le gouvernement du Canada peut retenir des impôts pouvant aller jusqu'à 25 %.\n\nSi vous vivez à l'extérieur du Canada pendant plus de six mois et que vous recevez le Supplément de revenu garanti (SRG), vous cesserez de recevoir cette prestation jusqu'à votre retour.\n\nVous ne pouvez pas percevoir l'allocation au survivant si vous êtes à l'étranger pendant plus de six mois.\n\nSi vous prévoyez de quitter le Canada pour une période de plus de six mois, vous devez [communiquer avec Service Canada](https://www.canada.ca/fr/contact/communiquez-avec-1-800-o-canada.html) pour éviter un plus-payé.",
+        "descriptionFr": "Le fait de passer du temps à l'étranger peut avoir des conséquences sur les prestations de la Sécurité de la vieillesse et du Régime de pensions du Canada (RPC) que vous recevez.\n\nVous pouvez continuer à recevoir vos pensions de l'État à l'étranger, mais le gouvernement du Canada peut retenir des impôts pouvant aller jusqu'à 25\u00a0%.\n\nSi vous vivez à l'extérieur du Canada pendant plus de six mois et que vous recevez le Supplément de revenu garanti (SRG), vous cesserez de recevoir cette prestation jusqu'à votre retour.\n\nVous ne pouvez pas percevoir l'allocation au survivant si vous êtes à l'étranger pendant plus de six mois.\n\nSi vous prévoyez de quitter le Canada pour une période de plus de six\u00a0mois, vous devez [communiquer avec Service Canada] (https://www.canada.ca/fr/contact/communiquez-avec-1-800-o-canada.html) pour éviter un plus-payé.",
```

Comment:
Non-breaking space `\u00a0` is not needed with `six mois`.

Reply:
remove `\u00a0` in front of `six mois.`